rpmio/rpmio.c

```diff
@@ -1134,10 +1139,15 @@ static rpmzstd rpmzstdNew(int fdno, const char *fmode)
 	}
 	nb = ZSTD_DStreamInSize();
     } else {					/* compressing */
-	if ((_stream = (void *) ZSTD_createCStream()) == NULL
-	 || ZSTD_isError(ZSTD_initCStream(_stream, level))) {
+	if ((_stream = (void *) ZSTD_createCCtx()) == NULL
+	 || ZSTD_isError(ZSTD_CCtx_setParameter(_stream, ZSTD_c_compressionLevel, level))) {
 	    goto err;
 	}
+
+	if (threads > 0)
+	  if (ZSTD_isError (ZSTD_CCtx_setParameter(_stream, ZSTD_c_nbWorkers, threads)))
```

Comment:
hmmpf, this always gives me an error on Fedora Rawhide:

```
warning: zstd library does not support multi-threading
```

With `ZSTD_getErrorName()` I get `Unsupported parameter`.

Reply:
Oh my, Fedora ships libzstd without MT support. ignore this one.